api/v1/client/src/commonMain/kotlin/api/RunsApi.kt

```diff
@@ -42,6 +43,33 @@ class RunsApi(
     suspend fun getOrtRun(id: Long): OrtRun =
         client.get("api/v1/runs/$id").body()
 
+    /**
+     * Download the logs of the run with the given [runId] and optional [level] and [steps]. The logs are streamed to
+     * the [streamTarget] function to avoid loading the whole log into memory.
+     * If [level] or [steps] are not provided, the server defaults are used.
+     */
+    suspend fun downloadLogs(
+        runId: Long,
+        level: LogLevel? = null,
+        steps: List<String> = emptyList(),
+        streamTarget: suspend (ByteReadChannel) -> Unit
+    ) {
+        val queryParameters = mutableListOf<String>().apply {
```

Comment:
Consider `listOfNotNull(level?.let { "level=${it.name}" }, ...)` or `buildList {}` instead.